ports/google-dawn/portfile.cmake

```
@@ -0,0 +1,51 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_git(
+		OUT_SOURCE_PATH SOURCE_PATH
+		URL https://dawn.googlesource.com/dawn
+		REF e3feea76729d52c050908e5f8588ee0b964112ff
+		PATCHES cmake_changes.diff absl_lts.diff src_tint_prefix.diff
```

Please modify all formats in this file. 
```suggestion
vcpkg_from_git(
	OUT_SOURCE_PATH SOURCE_PATH
	URL https://dawn.googlesource.com/dawn
	REF e3feea76729d52c050908e5f8588ee0b964112ff
	PATCHES cmake_changes.diff absl_lts.diff src_tint_prefix.diff
```